### PR TITLE
Add Host Logs README.md

### PR DESCRIPTION
**Pull request by chris-raitano**
- Added a README for windows host logs
- Updated README at root directory with windows host log file structure

### CODE REVIEW

**Review by cacarlt — APPROVED** [review]
Looks great!

File: README.md [hunk 1]
```diff
@@ -55,6 +55,9 @@ The general directory structure is:
 |   |   |   |── conf/                         - fluent, fluentbit and out_oms plugin configuration files
 |   |   |   |── scripts/                      - script files related to livenessproble, filesystemwatcher, keepCertificateAlive etc..
 |   |   |   |── certificategenerator/         - .NET code for the generation self-signed certificate of the windows agent
+|   |── hostologswindows/                     - scripts to build the cpp code for windows host logs
```
Comment (alrodrig1) [inline README.md]: typo? `hostologswindows`

File: kubernetes/windows/hostlogs/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+### Build Windows Host Logs code and Docker image
+```powershell
+cd %userprofile%\Docker-Provider\kubernetes\windows\hostlogs # based on your repo path
+docker login # if you want to publish the image to acr then login to acr via `docker login <acr-name>`
+powershell -ExecutionPolicy bypass  # switch to powershell if you are not on powershell already
+.\build-and-publish-docker-image.ps1 -image <acr uri>/<imagename>:<imagetag> # trigger build code and image and publish docker hub or acr
+```
+
+# Deploying WHL
+1. Open _Docker-Provider\kubernetes\host-logs-geneva.yaml_. Replace `VALUE_CONTAINER_IMAGE`, `VALUE_AKS_RESORUCE_ID`, `VALUE_AKS_CLUSTER_NAME`, and `VALUE_AKS_RESOURCE_REGION_VALUE` with the appropriate values
```
Comment (alrodrig1) [inline kubernetes/windows/hostlogs/README.md]: typo: `RESORUCE`

File: README.md [hunk 3]
```diff
@@ -55,6 +55,9 @@ The general directory structure is:
...
+|   |   |── Makefile.ps1                      - powershell script to build .net and go lang code and copy the files to amalogswindows directory
```
Comment (alrodrig1) [inline README.md]: Double-check for accuracy. This file seems to be only building the Liveness Probe, which is C++. Everything else seems to just be copied around.

**Comment by cacarlt** [comment]
[celebrate]     Cameron Carlton reacted to your message:
________________________________

**Pull request merged**